docs/sphinx/source/clearsky.rst

```diff
@@ -118,6 +116,66 @@ The Ineichen and Perez clear sky model parameterizes irradiance in terms
 of the Linke turbidity [Ine02]_. pvlib-python implements this model in
 the :py:func:`pvlib.clearsky.ineichen` function.
 
+
+The :py:func:`~pvlib.atmosphere.kasten96_lt` function can be used to calculate
```

Comment:
The "Turbidity data" section below introduces the ``clearsky.lookup_linke_turbidity`` function. Your example uses this function, so I think your new text/example should be appended to that section. Am I missing something?